src/System Application/App/Company Initialize/CompanyInitialize.Table.al

```diff
@@ -0,0 +1,33 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace System.Environment.Configuration;
+
+table 900 "Company Initialize"
+{
+    DataClassification = SystemMetadata;
+
+    fields
+    {
+        field(1; "Primary Key"; Integer)
```

Comment:
This should be moved to the company table so we track for each company.
Keep in mind that it is by design that we can run it multiple times.

Reply:
Can you explain why we want to run this multiple times?